pkg/deps/config/config.go

```diff
@@ -37,36 +42,129 @@ func SupplyConfig(
 }
 
 // NewSupplyEventsQueryClientFn returns a new function which constructs an
-// EventsQueryClient instance and returns a new depinject.Config which is supplied
-// with the given deps and the new EventsQueryClient.
+// EventsQueryClient instance and returns a new depinject.Config which is
+// supplied with the given deps and the new EventsQueryClient.
 func NewSupplyEventsQueryClientFn(
-	pocketNodeWebsocketUrl string,
+	pocketNodeWebsocketURL string,
```

Comment:
NIT: **TL;DR, this is technically, conventionally correct; however, I think we should prefer (dromedary) camel-casing acronyms for consistency for now but we should correct later.**

Thanks for taking notice of this @h5law! :raised_hands: While it would indeed be more conventional to use all caps for variable names (or parts thereof) consisting of acronyms, we seem to have gotten into a habit of ignoring this exception to the go variable naming convention extremely consistently thus far.

When I was pulling #129, I was reminded of our unconventional approach and figured that we'd have the opportunity to rectify it when we start [turning on linters](https://github.com/pokt-network/poktroll/pull/129/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9), specifically `govet` IIRC.

Reply:
Yeah my linter started complaining here lmao